federation/pkg/federation-controller/util/clusterselector/clusterselector.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterselector
+
+import (
+	"encoding/json"
+	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	federation_v1beta1 "k8s.io/kubernetes/federation/apis/federation/v1beta1"
+)
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster(clusterLabels map[string]string, annotations map[string]string) (bool, error) {
```

Comment:
Add a more meaningful comment since this is a public method.
Something like: "Parses the cluster selector annotation to find out if the object with that annotation should be forwarded to a cluster with the given clusterLabels."